sensor_msgs/msg/LinearAcceleration.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
+geometry_msgs/Vector3 linear_acceleration
+float64[9] linear_acceleration_covariance # Row major x, y z
```

Comment:
Maybe consider adding `bool is_gravity_compensated`? Defaulting to false sounds good as that is the old behavior.